frontend/src/components/Menu/Menu.js

```diff
@@ -30,6 +52,7 @@ function Menu() {
             <li className="Menu-list-item Menu-link">
               <Button to="/stores/new" label={t('merchantSignUp')} type="small"/>
             </li>
+            {isAuthenticated ? logoutField : loginField}
```

Comment:
wrap an `<li>` around this and remove it from the logout and loginField – keep it `DRY`